### DISCUSSION TITLE
Hierarchical Resources list

### DISCUSSION THREAD

**Discussion by najier** [vkurko/calendar#314] [Ideas]
Can you add this feature in the Resource column on the left hand side of the Timeline?

It is a Hierarchical list of Resource. 

Similar to https://bryntum.com/products/scheduler/examples/tasks/

A use case would be, say, there are employees doing work but they are grouped under different teams. When u expand a team it will show the schedule for each employee in that team. It is like a Group Scheduler.

**Comment by vkurko** [comment]
Just for reference #302.

**Comment by vkurko** [comment]
Nested resources have been implemented in v3.7.0.
